application/__init__.py

```diff
@@ -17,13 +17,21 @@
 #   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
```

Comment:
This is minor but PyCharm can automate for you:

Could you update the copyright line as
```
#  Copyright (c) 2021-2024 iCtrl Developers
```